IBAnimatable/BorderDesignable.swift

```diff
@@ -25,23 +25,18 @@ public protocol BorderDesignable {
 
 public extension BorderDesignable where Self: UIView {
   public func configBorder() {
-    func clearBorders() {
-      layer.sublayers?.filter  { $0.name == "borderSideLayer" || $0.name == "borderAllSides" }
+    // Clear borders
+    layer.sublayers?.filter  { $0.name == "borderSideLayer" || $0.name == "borderAllSides" }
         .forEach { $0.removeFromSuperlayer() }      
-    }
-    
+
     guard let unwrappedBorderColor = borderColor else {
-      clearBorders()
       return
     }
     
     if borderWidth.isNaN || borderWidth <= 0 {
-      clearBorders()
       return
     }
```

Comment:
I still think that we should remove that merge that `if` in the `guard ` using `where`. It just making the code bigger.